File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-02/non-plagiarized/05/T02.kt
package IR

import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    println("Enter the radius and length of a cylinder: ")
    val radlen = sc.nextLine()
    val splited = radlen.split("\\s+".toRegex())
    val rad = splited[0].toFloat()
    val len = splited[1].toFloat()
    println("The area is ${(rad * rad * Math.PI * 10000.0).roundToInt() / 10000.0}")
    println("The volume is ${(rad * rad * Math.PI * len * 10.0).roundToInt() / 10.0}")
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/non-plagiarized/11/T02.py
import math
from decimal import Decimal, ROUND_HALF_UP

def main():
    radius, length = map(float, input("Enter the radius and length of a cylinder: ").split())
    
    area = radius ** 2 * math.pi
    volume = area * length
    
    print(f"The area is {area:.4f}")
    print(f"The volume is {volume:.1f}")

if __name__ == "__main__":
    main()

